core/src/components/cat-tabs/cat-tabs.scss

```diff
@@ -26,9 +26,47 @@
 }
 
 .cat-tab {
-  padding: 0.5rem 0;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  transition: none;
+  padding: 1.125rem 0.75rem;
```

Comment:
The padding of "1.125rem 0.75rem" should be removed here and be applied to the cat-button instead

Reply:
fixed